TidepoolOnboarding/Assets.xcassets/Notification_1.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "notification_1.pdf",
```

Comment:
The filename should match the asset name, specifically "Notification_1" (note capitalization).